packages/xstate-graph/package.json

```diff
@@ -37,6 +41,15 @@
     "xstate": "^4.7.0"
   },
   "devDependencies": {
+    "@babel/cli": "^7.6.0",
```

Comment:
Seems like you have added those manually and you have not sorted those dependencies correctly.

Reply:
I'm not sure what this means. I tried:

<img width="419" alt="Screen Shot 2020-03-02 at 12 37 07 PM" src="https://user-images.githubusercontent.com/3596468/75706407-b5927680-5c82-11ea-8b5b-8328f5a57b29.png">

Would this mean I could
`yarn manypkg run @xstate/graph start`
and not need these deps in this package? I'll start trying things

Reply:
Unless you meant alphabetically. Just `:sort`'d them in case

Reply:
Sorry for not mentioning that - but yes, I was referring to the reported problem from `manypkg`. Glad you have found how to fix this using that tool 🎉